programming/minibot-assembly-tutorial.md

```diff
@@ -0,0 +1,101 @@
+# Minibot Assembly
+### Instructions on how to build your own mini robot!
+#### How to program your minibot is on the following tutorial. 
+
+Creating a minibot is great for testing small projects. It's an easy change to try out new parts out and costs considerably less than a larger robot. 
+
+------------------------------
+### PARTS NEEDED:
+
+We use many parts from Sainsmart's 4WD Robot Car Chasis Kit (https://www.sainsmart.com/products/4wd-robot-car-chasis-kit), along with some of our own senors, though you can also build entirely from scratch. 
```

Comment:
This is not autolinked. Can you make the text "Sainsmart's 4WD Robot Car Chassis Kit" link to the link you provided.

Also, "chassis" is spelled wrong. I realize that the website you linked to also spelled chassis wrong.